docs/source/releases/v3.2.rst

```diff
@@ -39,24 +43,61 @@ Minor changes
 
 - Added a new helper ``core.utils.is_ajax`` which replicates the logic of Django's ``HttpRequest.is_ajax``
   method that was deprecated in Django 3.1.
+  
 - Persisted ``OrderFactory.date_placed`` to the database when set, instead of just setting it on the object without saving it.
 
+- Fixed getting non public categories with ``get_categories`` by only returning browsable categories.
+
+- Reviews can now only be viewed and placed when the product is public. Before you could access the review urls if you knew them or they were indexed into a search engine.
+
+?- Remove title from shippingaddress
```

Comment:
I suggest we say something like "the `title` field has been removed by default from all address forms in Oscar. The database field is still present, and projects that wish to retain this field in their forms should add it back by overriding the `fields` property of the `Meta` class for applicable forms.